### PR TITLE
automl allow future time in feature

### PR DESCRIPTION
**Pull request by TheaperDeng**
related to issue #2946

### CODE REVIEW

File: pyzoo/zoo/automl/feature/time_sequence.py [hunk 1]
```diff
@@ -541,6 +534,8 @@ def _generate_features(self, input_df):
         df = input_df.copy()
         df["id"] = df.index + 1
 
+        cutoff_time = np.datetime64(np.max(df[self.dt_col]))
```
Comment (yushan111) [inline pyzoo/zoo/automl/feature/time_sequence.py:537]: df[self.dt_col].max().to_numpy() or just use np.inf?

**Review by yushan111 — COMMENTED** [review]
Others LGTM.

File: pyzoo/zoo/automl/feature/time_sequence.py (hunk 1 above)
Comment (TheaperDeng) [reply pyzoo/zoo/automl/feature/time_sequence.py:537]: np.inf doesn't work. I will try the other one.

**Comment by TheaperDeng** [comment]
Here is the related code in featuretools.
https://github.com/FeatureLabs/featuretools/blob/main/featuretools/computational_backends/calculate_feature_matrix.py#L172-L176
```python
        if cutoff_time is None:
            if entityset.time_type == NumericTimeIndex:
                cutoff_time = np.inf
            else:
                cutoff_time = datetime.now()
```

**Comment by TheaperDeng** [comment]
PR validation pass: http://10.239.47.210:18888/job/ZOO-PR-Validation/4463/

**Pull request merged**